docker/lite/Dockerfile.mysql57

```diff
@@ -51,7 +51,6 @@ ENV PATH $VTROOT/bin:$PATH
 # Copy artifacts from builder layer.
 COPY --from=builder --chown=vitess:vitess /vt/install /vt
 COPY --from=builder --chown=vitess:vitess /vt/src/vitess.io/vitess/web/orchestrator /vt/web/orchestrator
-COPY --from=builder --chown=vitess:vitess /vt/src/vitess.io/vitess/web/vtadmin /vt/web/vtadmin
```

Comment:
This reverts #9405 and probably breaks anyone currently relying on it.
Let's revert this one line for now.